dds/DCPS/RTPS/Spdp.cpp

```diff
@@ -915,6 +939,10 @@ Spdp::handle_auth_request(const DDS::Security::ParticipantStatelessMessage& msg)
 
   if (iter != participants_.end()) {
     if (msg.message_identity.sequence_number <= iter->second.auth_req_sequence_number_) {
+      if (DCPS::security_debug.auth_debug) {
+        ACE_DEBUG((LM_DEBUG, ACE_TEXT("(%P|%t) {auth_debug} DEBUG: Spdp::handle_auth_request() - ")
+                   ACE_TEXT("Dropped due to old sequence number\n")));
```

Comment:
Log the sequence numbers?

Reply:
How would a user meaningful use the sequence numbers?

Reply:
Would help to understand why the log message is there in the log